static/language-assets.js

```diff
@@ -15,6 +16,7 @@ var languageAssets = {
   // Spanish
   es: {
     continueButton: "Seguir",
+    joinButton: "Unirse a la cita",
```

Comment:
How was this generated?

Reply:
Google translate :)
We have not setup proper i18n support yet (and it is not very trivial).
I will wait for UX to weight in on the UI and string and then try to get a proof read of the translation.

Reply:
Checked with our tech writer and the string looks good.